src/frameworks/ez.md

```diff
@@ -1,21 +1,146 @@
 # eZ Platform
 
-eZ Platform is a Composer-based PHP CMS, and as such fits well with the Platform.sh model.  As a Symfony-based application its setup is very similar to Symfony.
+eZ Platform is a Composer-based PHP CMS, and as such fits well with the Platform.sh model.  As a Symfony-based 
+application its setup is very similar to Symfony.
 
-eZ Platform comes pre-configured for use with Platform.sh in version 1.13 and later and version 2.0.1 and later.  Those are the only versions that are supported.  Appropriate Platform.sh configuration files are included in the eZ Platform application itself, but of course may be modified to suit your particular site if needed.
+eZ Platform comes pre-configured for use with Platform.sh in version 1.13 and later. Version is 2.x and later is
+recommended.
+Those are the only versions that are supported.  Appropriate Platform.sh configuration files are included in the
```

Comment:
Is this supposed to be a new paragraph?  If so, it needs a blank line.  If not, remove the carriage return.

Reply:
done